### PR TITLE
[fix](bitmap) fix wrong result of bitmap_or for null

### PR DESCRIPTION
**Pull request by jacktengg**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Result of `select bitmap_to_string(bitmap_or(to_bitmap(1), null))` should be `1` instead `of null`.

This PR fix logic of bitmap_or and bitmap_or_count.

Other count related funcitons should also be checked and fix, they will be fixed in another PR.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run feut

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run p0

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run feut

**Comment by jacktengg** [comment]
run feut

**Comment by jacktengg** [comment]
run p0

**Comment by jacktengg** [comment]
run beut

**Comment by jacktengg** [comment]
run feut

**Comment by jacktengg** [comment]
run clickbench

File: gensrc/script/doris_builtins_functions.py [hunk 1]
```diff
@@ -1407,8 +1407,8 @@
     [['bitmap_count'], 'BIGINT', ['BITMAP'], 'ALWAYS_NOT_NULLABLE'],
     [['bitmap_and_not_count'], 'BIGINT', ['BITMAP','BITMAP'], ''],
     [['bitmap_empty'], 'BITMAP', [], 'ALWAYS_NOT_NULLABLE'],
-    [['bitmap_or'], 'BITMAP', ['BITMAP','BITMAP','...'], ''],
-    [['bitmap_or'], 'BITMAP', ['BITMAP','BITMAP'], ''],
+    [['bitmap_or'], 'BITMAP', ['BITMAP','BITMAP','...'], 'CUSTOM'],
```
Comment (HappenLee) [inline gensrc/script/doris_builtins_functions.py:1410]: change a function nullable way may cause coredump in update

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run buildall

**Comment by jacktengg** [comment]
run p0

File: be/src/vec/functions/function_bitmap_variadic.cpp [hunk 2]
```diff
@@ -113,13 +197,27 @@ class FunctionBitMapVariadic : public IFunction {
                 std::conditional_t<is_complex_v<ResultType>, ColumnComplexType<ResultType>,
                                    ColumnVector<ResultType>>;
         typename ColVecResult::MutablePtr col_res = nullptr;
+
+        typename ColumnUInt8::MutablePtr col_res_nulls;
+        auto& result_info = block.get_by_position(result);
+        // special case for bitmap_or and bitmap_or_count
+        if (!use_default_implementation_for_nulls() && result_info.type->is_nullable()) {
+            col_res_nulls = ColumnUInt8::create(input_rows_count);
```
Comment (HappenLee) [inline be/src/vec/functions/function_bitmap_variadic.cpp:205]: ```col_res_nulls = ColumnUInt8::create(input_rows_count, false);```

File: be/src/vec/functions/function_bitmap_variadic.cpp [hunk 3]
```diff
@@ -54,18 +96,39 @@ namespace doris::vectorized {
...
+                }                                                                                  \
+                const auto& col_data = *col_data_ptr;                                              \
                 for (size_t row = 0; row < input_rows_count; ++row) {                              \
                     vals[row] OP col_data[row];                                                    \
                 }                                                                                  \
             }                                                                                      \
             for (size_t row = 0; row < input_rows_count; ++row) {                                  \
                 res[row] = vals[row].cardinality();                                                \
             }                                                                                      \
+            if (res_nulls) {                                                                       \
```
Comment (HappenLee) [inline be/src/vec/functions/function_bitmap_variadic.cpp:126]: delete the code. better init the ColumnUInt8 by constructor

Comment (HappenLee) [inline be/src/vec/functions/function_bitmap_variadic.cpp:126]: delete the code. better init the ColumnUInt8 by constructor

**Comment by jacktengg** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**